File: src/main/java/no/nav/fo/veilarbregistrering/bruker/resources/KontaktinfoDto.java
package no.nav.fo.veilarbregistrering.bruker.resources;


public class KontaktinfoDto {

    private String telefonnummerHosKrr;
    private String telefonnummerHosNav;

    public KontaktinfoDto() {
    }

    public String getTelefonnummerHosKrr() {
        return this.telefonnummerHosKrr;
    }

    public String getTelefonnummerHosNav() {
        return this.telefonnummerHosNav;
    }

    public KontaktinfoDto setTelefonnummerHosKrr(String telefonnummerHosKrr) {
        this.telefonnummerHosKrr = telefonnummerHosKrr;
        return this;
    }

    public KontaktinfoDto setTelefonnummerHosNav(String telefonnummerHosNav) {
        this.telefonnummerHosNav = telefonnummerHosNav;
        return this;
    }

}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/bruker/resources/KontaktinfoDto.kt
package no.nav.fo.veilarbregistrering.bruker.resources

import no.nav.fo.veilarbregistrering.bruker.Navn

data class KontaktinfoDto(
    val telefonnummerHosKrr: String? = null,
    val telefonnummerHosNav: String? = null,
    val navn: Navn? = null,
)